app/javascript/shared/wellHelpers.spec.js

```diff
@@ -15,4 +15,29 @@ describe('wellHelpers', () => {
       expect(findUniqueIndex([5, 1, 2, 1, 2, 3], 5)).toBe(0)
     })
   })
+
+  describe('requestsForWell', () => {
+    const request1 = { uuid: '1' }
+    const request2 = { uuid: '2' }
+
+    const well = {
+      uuid: '3',
+      position: { name: 'A1' },
+      aliquots: [
+        {
+          uuid: '4',
+          request: request1,
+        },
+        {
+          uuid: '5',
+          request: null,
```

Comment:
not quite following why this is null
is it possible for one aliquot in a well to have a request and the other not?
also, is it possible for request1 to be both in requests_as_source (which it will be if the submission is made on this labware) AND in the aliquot (which is set only on children of the submitted well)?
maybe add some comments describing the scenario?

Reply:
the real scenario was more that multiple aliquots in the well has the same aliquot.request yes?

Reply:
You're right, it's not representing the real scenario, I was just trying to cover everything in one test (bad practice 😅 ). I'll split them out.

I don't think it's expected for the same request to be in both requests_as_source and on the aliquot... but I wouldn't be surprised if that happens in the future (or in some edge case now), so I thought I should account for it.

Reply:
requestsForWell filters falsy values out and the test is describing the requestsForWell function. Having a null request covers that case.